microsoft/testsuites/dpdk/dpdknffgo.py

```diff
@@ -0,0 +1,130 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from typing import List, Type
+
+from lisa.executable import Tool
+from lisa.operating_system import Debian
+from lisa.tools import Echo, Gcc, Git, Make, Tar, Wget
+from lisa.util import UnsupportedDistroException
+
+
+class DpdkNffGo(Tool):
+
+    NFF_GO_SRC_LINK = "https://github.com/intel-go/nff-go.git"
+    REPO_DIR = "nff-go"
+    GO_TAR = "go1.17.6.linux-amd64.tar.gz"
```

Comment:
Can we create a tool for go?

Reply:
We can but I'll do it in a new PR once this is merged